src/js/containers/keyword/table/ResultsTableContainer.jsx

```diff
@@ -0,0 +1,323 @@
+        internal: 'contracts'
+    },
+    {
+        label: 'Grants',
+        internal: 'grants'
+    },
+    {
+        label: 'Direct Payments',
+        internal: 'direct_payments'
+    },
+    {
+        label: 'Loans',
+        internal: 'loans'
+    },
+    {
+        label: 'Other',
+        internal: 'other'
+    }
+];
+
+export default class ResultsTableContainer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            tableType: 'contracts',
+            page: 1,
+            lastPage: true,
+            counts: {},
+            sort: {
+                field: 'Transaction Amount',
+                direction: 'desc'
+            },
+            columns: {},
+            inFlight: false,
+            error: false,
+            results: [],
+            tableInstance: `${uniqueId()}` // this will stay constant during pagination but will change when the keyword or table type changes
+        };
+
+        this.searchRequest = null;
+        this.tabCountRequest = null;
+
+        this.switchTab = this.switchTab.bind(this);
+        this.loadNextPage = this.loadNextPage.bind(this);
+        this.updateSort = this.updateSort.bind(this);
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps.keyword !== this.props.keyword) {
+            // filters changed, update the search object
+            this.loadColumns();
+            this.pickDefaultTab();
+        }
+    }
+
+    pickDefaultTab() {
+        // get the transaction counts for the current filters
+        if (this.tabCountRequest) {
+            this.tabCountRequest.cancel();
+        }
+
+        this.setState({
+            inFlight: true,
+            error: false
+        });
+
+        const filters = {
+            keyword: this.props.keyword
+        };
+
+        this.tabCountRequest = KeywordHelper.performTabCountSearch({
+            filters
+        });
+
+        this.tabCountRequest.promise
+            .then((res) => {
+                this.parseTabCounts(res.data);
+            })
+            .catch((err) => {
+                if (!isCancel(err)) {
+                    this.setState({
+                        inFlight: false,
+                        error: true
+                    });
+
+                    console.log(err);
+                }
+            });
+    }
+
+    parseTabCounts(data) {
+        const transactionCounts = data.results;
+        let firstAvailable = '';
+
+        // Set the first available award type to the first non-zero entry in the
+        for (let i = 0; i < tableTypes.length; i++) {
+            const tableType = tableTypes[i].internal;
+
+            if (transactionCounts[tableType] > 0) {
+                firstAvailable = tableType;
+                break;
+            }
+        }
+
+        // If none of the award types are populated, set the first available tab to be the
+        // first tab in the table
+        if (firstAvailable === '') {
```

Comment:
Can you change this equality check to a falsy string check (`if (!firstAvailable)`)? There are also a few other places this happens in this container.